### DISCUSSION TITLE
Chat crashing

### DISCUSSION THREAD

**Discussion by xavierlvll** [Pryaxis/TShock#2887] [Q&A]
So this has been around for years, but recently becoming a bigger issue, anyway to stop it
here is an example on what people paste to crash people ╚╚╚╚╚╚╚╚╚╚╚╚╚┐b\1eú╝ÖbÖ╚┴b:wlen»l6l│b ╡v╣1h½b│1h└╚╚╚╚╚╚╚╚╚╚╚╚╚╚┐b\1eú╝ÖbÖ╚┴b:wlen»l6l│b ╡v╣1h½b│1h└╚
This Really only effects low end computer's and Mobile phones

**Comment by sgkoishi** [comment]
Can confirm.
PC send chat `╚╚╚╚╚╚╚╚╚╚╚╚╚┐b\1eú╝ÖbÖ╚┴b:wlen»l6l│b ╡v╣1h½b│1h└╚╚` is ok, but `/bc "╚╚╚╚╚╚╚╚╚╚╚╚╚┐b\1eú╝ÖbÖ╚┴b:wlen»l6l│b ╡v╣1h½b│1h└╚╚╚╚╚╚╚╚╚╚╚╚╚╚┐b\1eú╝ÖbÖ╚┴b:wlen»l6l│b ╡v╣1h½b│1h└╚"` breaks mobile (Android).

Can you find any crash log on PC?

**Reply by xavierlvll** [reply]
I do not have any logs of it happening on PC, and I can not recreate it
